packages/comparator/src/type.ts

```diff
@@ -201,16 +256,28 @@ function compare(src: Type, dst: Type, rule?: Rule): TypeComparison {
     if (diffRatio < RATIO_THRESHOLD || diffCount < COUNT_THRESHOLD) {
       score = commonCount / maxLength
     }
+    const diff = jsonDiff.diff(src, dst)
+    const diffString = jsonDiff.diffString(src, dst, {
+      full: true,
+      color: false,
+    })
     const match = score === 1
     return match
       ? { ...cmp, score }
-      : { ...cmp, score, dstValue: stringify(dst) }
+      : { ...cmp, score, dstValue: stringify(dst), diff, diffString }
   }
 
+  const diff = jsonDiff.diff(src, dst)
+  const diffString = jsonDiff.diffString(src, dst, {
+    full: true,
+    color: false,
+  })
   return {
     ...cmp,
     dstType: getTypeName(dst),
-    dstValue: stringify(dst)
+    dstValue: stringify(dst),
+    diff,
+    diffString,
```

Comment:
I don't know what's happened to me but lately, I've become a big fan of reducing local variables and assignments. What do you think about the following?
```suggestion
    dstValue: stringify(dst),
    diff: diff(src, dst),
    diffString: diffString(src, dst, {full: true, color: false})
```